### ISSUE TITLE
Create functions that allows go get empty/default uuid

### ISSUE THREAD

**Issue by EvilLord666** [satori/go.uuid#117]
Is it possible to have function `uuid.Empty()` or `uuid.Default()` which returns uuid that is equal to 00000000-0000-0000-0000-000000000000?

**Comment by fox091** [comment]
@EvilLord666 You should not be using this repo any more.  It has a major vulnerability and is no longer being maintained.  See #103 for a chain of issues and details.

**Comment by EvilLord666** [comment]
@fox091 Maybe this repo should be archived?

**Comment by fox091** [comment]
@EvilLord666 Yes it should be archived.  Unfortunately the maintainer hasn't been active for over 3 years, so there's nothing we can do.
